panther_lights_controller/README.md

```diff
@@ -14,37 +14,173 @@ Package used to control the Husarion Panther LED panels.
 
 #### Parameters
 
-- `~animations` [*list*]: it is a required ROS parameter containing a list of defined animations.
-- `~global_brightness` [*float*, default: **1.0**]: LED global brightness.
-- `~num_led` [*int*, default: **46**]: number of LEDs in single panel.
+- `~animations` [*list*, default: None]: it is a required ROS parameter containing a list of defined animations.
 - `~controller_frequency` [*float*, default: **100**]: frequency at which the lights controller node will process animations.
-- `~test` [*bool*, default: **false**]: allows testing mode with some extra functionalities.
+- `~global_brightness` [*float*, default: **1.0**]: LED global brightness. Range between [0,1].
+- `~num_led` [*int*, default: **46**]: number of LEDs in single panel.
+- `~test` [*bool*, default: **false**]: enables testing mode with some extra functionalities.
+- `~user_animations` [*list*, default: None]: it is an optional ROS parameter containing a list of animations defined by the user.
 
 ## Animations
 
-Basic animations are parsed as a list using the ROS parameter. They are loaded on node start directly from `config/panther_lights_animations.yaml`. Supported keys are:
+Basic animations are parsed as a list using the ROS parameter. Default animations supplied by Husarion are listed in table below.
+
+| ID  | NAME              | DESCRIPTION                                                 |
+| --- | ----------------- | ----------------------------------------------------------- |
+| 0   | E_STOP            | red expanding from the center to the edges                  |
+| 1   | READY             | green expanding from center to the edges                    |
+| 2   | ERROR             | red blinking twice                                          |
+| 3   | MANUAL_ACTION     | blue expanding from the center to the edges                 |
+| 4   | AUTONOMOUS_ACTION | orange expanding from center to the edges                   |
+| 5   | GOAL_ACHIEVED     | purple blinking three times                                 |
+| 6   | LOW_BATTERY       | two orange stripes moving towards the center, repeats twice |
+| 7   | CRITICAL_BATTERY  | two red stripes moving towards the center, repeats twice    |
 
+Default animations are described and loaded on the node start, directly from `config/panther_lights_animations.yaml`. Supported keys are:
+
+- `animation` [*dict*]: definition of animation. See section below for more info.
 - `id` [*int*]: ID of an animation.
-- `animation` [*dict*]: description of animation.
-- `name` *(optional)* [*string*]: name of an animation.
-- `interrupting` *(optional)* [*bool*]: if *true* animation will interrupt currently displayed animation.
+- `interrupting` [*bool*, optional]: if *true* animation will interrupt currently displayed animation.
+- `name` [*string*, optional]: name of an animation.
 
 ### ImageAnimation
```

Comment:
There is rough transition between default animations and animation class. An additional subsection saying something like "AnimationTypes" or something like this would give the needed introduction to the topic